mods/default/nodes.lua

```diff
@@ -183,6 +183,9 @@ Corals
 default:coral_brown
 default:coral_orange
 default:coral_skeleton
+default:coral_green
+default:coral_pink
+default:coral_cyan
```

Comment:
Ideally these could be moved to before brown, orange, skeleton to match the order of registration.

Reply:
Done

Reply:
Great, very quick =)